A32NX/html_ui/Pages/A32NX_Core/A32NX_FWC.js

```diff
@@ -286,4 +309,80 @@ class A32NX_FWC {
         this.ldgMemo = showInApproach || invalidRadioMemo || this.flightPhase === 8 || this.flightPhase === 7;
         SimVar.SetSimVarValue("L:A32NX_FWC_LDGMEMO", "Bool", this.ldgMemo);
     }
+
+    altitudeWarning(_deltaTime) {
+        const indicatedAltitude = Simplane.getAltitude();
+        SimVar.SetSimVarValue("L:A32NX_ALT_DEVIATION_SHORT", "Bool", false);
+        SimVar.SetSimVarValue("L:A32NX_ALT_DEVIATION", "Bool", false);
```

Comment:
Although I'm not fully aware how the button and short sound booleans work, the use of transitions and states could prevent an excessive reset of these deviation booleans on every frame update.